draft-birkholz-rats-corim.md

```diff
@@ -132,17 +353,436 @@ concise-mid-tag = {
+$tag-rel-type-choice /= comid.patches
+~~~
+
+The following describes each member of the linked-tag-map container.
+
+comid.linked-tag-id:
+
+: The tag-id of the linked tag. A linked tag MAY be a CoMID tag or a CoSWID tag.
+
+comid.tag-rel:
+
+: The relationship type with the linked tag. The relationship type MAY be `include`, `or-includes`, `supplements`, `updates`, `replaces`, or `patches`, as well as other types well-defined by additional specifications. 
+
+{: #model-triples-map}
+## The triples-map Container
+
+A set of directed properties that associate sets of data to provide reference values, endorsed values, verification key material or identifying key material for a specific hardware module that is a component of a composite device. The map provides the core element of CoMID tags that associate remote attestation relevant data with a distinct hardware component that runs an execution environment (a module that is either a Target Environment and/or an Attesting Environment). This rule and its constraints MUST be followed when generating or validating a CoMID tag.
+
+~~~ CDDL
+triples-map = non-empty<{
+  ? comid.reference-triples => one-or-more<reference-triple-record>
+  ? comid.endorsed-triples => one-or-more<endorsed-triple-record>
+  ? comid.attest-key-triples => one-or-more<attest-key-triple-record>
+  ? comid.identity-triples => one-or-more<identity-triple-record>
+  * $$triples-map-extension
+}>
+~~~
+
+The following describes each member of the triple-map container.
+
+comid.reference-triples:
+
+: A directed property that associates reference measurements with a module that is a Target Environment.
+
+comid.endorsed-triples:
+
+: A directed property that associates endorsed measurements with a module that is a Target Environment or Attesting Environment.
+
+comid.attest-key-triples:
+
+: A directed property that associates key material used to verify evidence generated from a module that is an attesting environment.
+
+comid.identity-triples:
+
+: A directed property that associates key material used to identify a module instance or a module class that is an identifying part of a device(-set).
+
+$$triples-map-extension:
+
+: This CDDL socket is used to add new information elements to the triples-map container. See FIXME.
+
+{: #model-environment-map}
+## The environment-map Container
+
+This map represents the module(s) that a triple-map can point directed properties (relationships) from in order to associate them with external information for remote attestation, such as reference values, endorsement and endorsed values, verification key material for evidence, or identifying key material for module (re-)identification. This map can identify a single module instance via `comid.instance` or groups of modules via `comid.group`. Referencing classes of modules requires the use of the more complex `class-map` container. This rule and its constraints MUST be followed when generating or validating a CoMID tag.
```

Comment:
The document and text looks great! Few minor comments here!

1) Kindly introduce class concept to the reader first and then instance and group concept.

2) Perhaps the class definition can be slightly simplified, on the following lines:
This map can identify a either a module class which identifies a device family  via `comid.class`  or a single module instance via via `comid.instance` or groups of modules via `comid.group`.